develop/TOOLS/DIS/Troubleshooting/DIS_Troubleshooting_DisInject.md

```diff
@@ -0,0 +1,58 @@
+---
+uid: DIS_Troubleshooting_DisInject
+keywords: dis inject issue, dis inject slscripting
+---
+
+# DIS Inject problems
+
+## Breakpoints are not hitting
+
+### Possible solutions
+
+- Projects need to be built in [Debug mode](https://learn.microsoft.com/en-us/visualstudio/debugger/what-is-debugging?view=vs-2022#debug-mode-vs-running-your-app).
+
+  DIS will run the code, so you only need to make sure the configuration is set to *Debug*.
+
+- Multiple SLScripting processes: Starting from [DIS 2.44](xref:DIS_2.44#dis-inject-attaching-the-visual-studio-debugger-to-multiple-slscripting-processes-id_37042), DIS will attach to all SLScripting processes.
```

Comment:
I'd prefer to phrase this as a solution too. Do I interpret it correctly like this?

```suggestion
- In case multiple SLScripting processes are used, make sure you use [DIS 2.44](xref:DIS_2.44#dis-inject-attaching-the-visual-studio-debugger-to-multiple-slscripting-processes-id_37042) or higher, as DIS will then attach to all SLScripting processes.
```